rules/ruleset.go

```diff
@@ -52,9 +52,9 @@ var (
 	errRuleSetNotTombstoned = errors.New("ruleset is not tombstoned")
 	errRuleNotFound         = errors.New("rule not found")
 	errNoRuleSnapshots      = errors.New("rule has no snapshots")
-	errUnknownOpType        = merrors.NewInvalidInputError("changes must contain an op")
 	ruleActionErrorFmt      = "cannot %s rule %s"
 	ruleSetActionErrorFmt   = "cannot %s ruleset %s"
+	unknownOpTypeFmt        = "unknown op type %v, op"
```

Comment:
Hm do you need the `, op`? Also you can probably just inline this.

Reply:
I was a little confused by the `, op` as well. You suggested it above but it may have been a typo. 

As for inlining it is used in two places on line 923 and 948 and it seems like the convention of the repo that if there is an error template string that is used more than once it is put up top as a constant.